Exercise_1.py

```diff
@@ -1,19 +1,50 @@
+"""
+// Time Complexity : for show() - O(n)
```

Comment:
It's a good practice to define what 'n' is according to the problem. eg- Space: O(n) where n is the size of the stack